docs/UserGuide.md

```diff
@@ -348,9 +352,24 @@ _Module prefix refers to the first two characters of every module name._
 **Q**: How do I transfer my data to another Computer?<br>
 **A**: Install Mass Linkers in the other computer and overwrite the empty data file it creates with the file that contains the data of your previous Mass Linkers home folder.
 
+**Q**: Do I need internet connection to use Mass Linkers?<br>
+**A**: No, internet connection is not required. 
+
+**Q**: I am using Mac and I tried opening the `MassLinkers.jar` file by double-clicking it in Finder. It says *"MassLinkers.jar" cannot be opened because it is from an unidentified developer.*<br>
+**A**: Right click the `MassLinkers.jar` file and select `Open with JavaLauncher(default)`.
```

Comment:
Maybe we can say:

Right click the `MassLinkers.jar` file and select `Open`. When a warning message that says _macOS cannot verify the developer of "MassLinkers.jar". Are you sure you want to open it?_ appears, select `Open`. 

Alternatively, you can right click the `MassLinkers.jar` file and select `Open with JavaLauncher(default)`". When the same warning message as the method above appears, select `Open`.